frontend/src/components/Header/Header.tsx

```diff
@@ -9,10 +9,9 @@ import * as screenfull from 'screenfull'
 import styles from '../../styling'
 
 export const Header: React.FC = () => {
-  const { device, admin, user } = useSelector((state: ApplicationState) => ({
+  const { device, guest } = useSelector((state: ApplicationState) => ({
+    guest: state.backend.admin && state.auth.user && state.auth.user.username !== state.backend.admin,
```

Comment:
Suggestion: maybe this should just be a computed value in the model?

Reply:
What is this "Computed value" in the model you speak of?